pages/dkp/konvoy/2.0/cli-tools/helm/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: layout.pug
+navigationTitle: Helm CLI
+title: Helm CLI
+menuWeight: 5
+excerpt: Helm CLI Usage
+beta: false
+enterprise: false
+---
+
+This section describes how to install the Helm CLI. Helm can be installed either from source or from pre-built binary releases. This topic focuses on installing the latest Helm binaries, which is known as Helm 3.
+
+You may encounter some situations where you need to use an older version of Helm, known as Helm 2. If so, see the Helm documentation.
+
+## What Is Helm
```

Comment:
No need for this. let's just add a link to their docs.

Reply:
Do you mean like changing this section to something like:

```
## What Is helm

Have a look at their documentation [link]
```

Or do you mean this page as a whole?

Reply:
The first option. Let's rely on Helm to describe what they do.